recipes/pypgatk/meta.yaml

```diff
@@ -0,0 +1,115 @@
+{% set name = "pypgatk" %}
+{% set version = "0.0.4" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: "c249729dcd9c344f88469826a7d023cd4ada3236172832fa90a950208b8576da"
+
+build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
+
+requirements:
+  host:
+    - argcomplete ==1.9.5
+    - argh ==0.26.2
+    - asn1crypto ==0.24.0
+    - astroid ==2.2.5
+    - bcrypt ==3.1.6
+    - biopython ==1.73
+    - certifi ==2019.3.9
+    - cffi ==1.12.3
+    - chardet ==3.0.4
+    - click ==7.0
+    - cryptography ==2.6.1
+    - gffutils ==0.9
+    - idna ==2.8
+    - isort ==4.3.18
+    - lazy-object-proxy ==1.4.0
+    - mccabe ==0.6.1
+    - numpy ==1.16.3
+    - paramiko ==2.4.2
+    - pip
+    - pyasn1 ==0.4.5
+    - pycparser ==2.19
+    - pyfaidx ==0.5.5.2
+    - pylint ==2.3.1
+    - pynacl ==1.3.0
+    - pysftp ==0.2.9
+    - python >=3.6
+    - pyvcf ==0.6.8
+    - pyyaml ==5.1
+    - requests ==2.21.0
+    - simplejson ==3.16.0
+    - six ==1.12.0
+    - typed-ast ==1.3.5
+    - urllib3 ==1.24.2
+    - wrapt ==1.11.1
+  run:
+    - argcomplete ==1.9.5
+    - argh ==0.26.2
+    - asn1crypto ==0.24.0
+    - astroid ==2.2.5
+    - bcrypt ==3.1.6
+    - biopython ==1.73
+    - certifi ==2019.3.9
+    - cffi ==1.12.3
+    - chardet ==3.0.4
+    - click ==7.0
+    - cryptography ==2.6.1
+    - gffutils ==0.9
+    - idna ==2.8
+    - isort ==4.3.18
+    - lazy-object-proxy ==1.4.0
+    - mccabe ==0.6.1
+    - numpy ==1.16.3
+    - paramiko ==2.4.2
+    - pyasn1 ==0.4.5
+    - pycparser ==2.19
+    - pyfaidx ==0.5.5.2
+    - pylint ==2.3.1
+    - pynacl ==1.3.0
+    - pysftp ==0.2.9
+    - python >=3.6
+    - pyvcf ==0.6.8
+    - pyyaml ==5.1
+    - requests ==2.21.0
+    - simplejson ==3.16.0
+    - six ==1.12.0
+    - typed-ast ==1.3.5
+    - urllib3 ==1.24.2
+    - wrapt ==1.11.1
+
+test:
+  imports:
+    - pypgatk
+    - pypgatk.cgenomes
+    - pypgatk.commands
+    - pypgatk.db
+    - pypgatk.ensembl
+    - pypgatk.proteomics
+    - pypgatk.proteomics.db
+    - pypgatk.tests
+    - pypgatk.toolbox
+  commands:
+    - which pypgatk_cli.py
+    - pypgatk_cli.py -h
+
+
+about:
+  home: "http://github.com/bigbio/py-pgatk"
+  license: "Apache 2"
+  license_family: "APACHE"
+  license_file:
```

Comment:
add `LICENSE` here.

Reply:
I added the link to the license but one of you tests complain about it because it was not in the same folder than meta.yaml. I will check for an example and add it.